libr/cons/input.c

```diff
@@ -230,6 +230,13 @@ R_API int r_cons_arrow_to_hjkl(int ch) {
 	return ch;
 }
 
+static inline void flush_istream(FILE *in_stream) {
+	int ch;
+	while ((ch = fgetc (in_stream)) != '\n' && ch != EOF) {
```

Comment:
imho the fflush(stdin) should be nonblock. i'm still unsure if that's the behaviour we are expecting. cc @ret2libc

Reply:
I didn't dig too much into this PR, though I've seen that it doesn't work as it is and I've provided ways to test the actual code that is being changed. With default parameters, you don't even reach this code usually. I'm ok with replacing `fflush(stdin)`, though this does not seem the right way to do it :(